# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1305469454 25200
#      Sun May 15 07:24:14 2011 -0700
# Node ID 72e60cf50dce3be3fa021ed58f831b70e3738177
# Parent  48b9e8344a03c01a775f1e80e6f98a99c0b60d7d
nonzeros.m: Ignore unused return arguments from find().

* nonzeros.m: Ignore unused return arguments from find().

diff --git a/scripts/sparse/nonzeros.m b/scripts/sparse/nonzeros.m
--- a/scripts/sparse/nonzeros.m
+++ b/scripts/sparse/nonzeros.m
@@ -22,18 +22,19 @@
 ## @end deftypefn
 
 function t = nonzeros (s)
 
   if (nargin != 1)
     print_usage ();
   endif
 
-  [i, j, t] = find (s);
+  [~, ~, t] = find (s);
 
   t = t(:);
 
 endfunction
 
+
 %!assert(nonzeros([1,2;3,0]),[1;3;2])
 %!assert(nonzeros([1,2,3,0]),[1;2;3])
 %!assert(nonzeros(sparse([1,2;3,0])),[1;3;2])
 %!assert(nonzeros(sparse([1,2,3,0])),[1;2;3])
